### PR TITLE
made index.html homepage & menuitem

### PR DESCRIPTION
**Pull request by news-papers**
I made "index.html" a homepage and took out digitised from all

### CODE REVIEW

**Comment by news-papers** [comment]
Don't know if index.html is designed to be a homepage at all but it's better then nothing

**Review by nznewspapers — COMMENTED** [review]
I think we should wait on this. I don't like saying "index.html" in a link. Let's discuss tomorrow.

File: docs/index.html [hunk 1]
```diff
@@ -59,6 +59,6 @@ <h2 id="pagetitle">New Zealand Newspapers</h2>
     </div>
 
     <script src="html-functions.js"></script>
-    <script src="titles.js"></script>
+    <script src="index.js"></script>
```
Comment (nznewspapers) [inline docs/index.html:62]: I want to wait to make this change until I can fix the homepage.

**Review by nznewspapers — CHANGES_REQUESTED** [review]
We don't need this change.

**Pull request closed without merging**